packages/@elements/search-driver-plugin/package.json

```diff
@@ -19,12 +19,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/groupby/sfx-logic.git",
-    "directory": "packages/@sfx/search-driver-plugin"
+    "url": "https://github.com/groupby/elements-logic.git",
+    "directory": "packages/@elements/search-driver-plugin"
   },
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@sfx/events": "groupby/sfx-events#c4a4853a424992db9c170fea6990dc2a52c3061e"
+    "@elements/events": "groupby/elements-events#c4a4853a424992db9c170fea6990dc2a52c3061e"
```

Comment:
question does this hash change at this point? Or do we expect it to change once we update the events repo?

Reply:
I used yarn link to use the PR that introduces the renaming for now. Once that is merged in, I will be updating the hash. (if I'm remembering the process correctly)